app/views/users/_user.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= image_tag(@user.photo, :size => "150x150") %>
```

Comment:
- [ ] Please kindly fix the migration error as requested by the previous code reviewer, in your db/schema.rb kindly update lines  39, 40 and 51 as we have below 
```
    t.integer "comments_counter" default 0
    t.integer "likes_counter" default 0
```
To 
```
    t.integer "comments_counter", default: 0
    t.integer "likes_counter", default: 0
```